src/thumbnailer_near_lossless.cc

```diff
@@ -222,6 +216,12 @@ Thumbnailer::Status Thumbnailer::NearLosslessEqual(WebPData* const webp_data) {
       *webp_data = new_webp_data;
     } else {
       WebPDataClear(&new_webp_data);
+      for (int curr_ind : near_ll_frames) {
+        frames_[curr_ind].config.near_lossless = 0;
+        CHECK_THUMBNAILER_STATUS(
+            GetPictureStats(curr_ind, &frames_[curr_ind].encoded_size,
```

Comment:
This re encodes all the frames, which is extra work, right? Maybe we can save the previous stats instead and restore them? (you can add a TODO comment to fix this in another commit if you want)

Reply:
I resolved this issue in this PR too because it it not complicate. 
PTAL.